omg/mechanics/movement.py

```diff
@@ -42,41 +47,59 @@ def on_key_press(self, key, modifiers):
         """Called when the user presses a key."""
         if key == self.key_forward:
             self.pressed_forward = True
+            self.move_direction = (self.move_direction[0], 1)
         elif key == self.key_backward:
             self.pressed_backward = True
+            self.move_direction = (self.move_direction[0], -1)
         elif key == self.key_left:
             self.pressed_left = True
+            self.move_direction = (-1, self.move_direction[1])
         elif key == self.key_right:
             self.pressed_right = True
+            self.move_direction = (1, self.move_direction[1])
 
-        [self.change_direction_x, self.change_direction_y] = (
-            self._calculate_displacement_directions()
-        )
+        # [self.change_direction_x, self.change_direction_y] = (
+        #     self._calculate_displacement_directions()
+        # )
 
     def on_key_release(self, key, modifiers):
         """Called when the user releases a key."""
         if key == self.key_forward:
             self.pressed_forward = False
+            if not self.pressed_backward:
+                self.move_direction = (self.move_direction[0], 0)
```

Comment:
Tuples are immutable objects in Python. Therefore, you need to set entire tuple whenever you need to change the tuple. Then, my question is: can we use another data structure such as `List` here?